modules/migrate-sdn-ovn-osd.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="migrate-sdn-ovn-ocm-cli_{context}"]
+= Initiate migration using the OCM CLI
+
+[WARNING]
+====
+You can only initiate migration on clusters that are version 4.15.22 and above or 4.16.24 and above.
+====
+
+To initiate the migration, run the following post request:
+[source,terminal]
+----
+$ ocm post /api/clusters_mgmt/v1/clusters/{cluster_id}/migrations <1>
+----
+<1> Replace `<cluster_id>` with the ID of the cluster you want to migrate to the OVN-Kubernetes network plugin.
+
+The post request will respond with a url to the created migration.
```

Comment:
I'm not sure how detailed we want to be here, but _technically_ the API response will be the entire created object (which does indeed contain a url to itself). Example:
```
$ ocm post /api/clusters_mgmt/v1/clusters/2gaog4ns6phso6q17v07rj3rre9dm24e/migrations --body test_body_migration.json
{
  "kind":"ClusterMigration",
  "href":"/api/clusters_mgmt/v1/clusters/2gaog4ns6phso6q17v07rj3rre9dm24e/migrations/2gau4ougs26d231o11vv51400s8akkbu",
  "id":"2gau4ougs26d231o11vv51400s8akkbu",
  "cluster_id":"2gaog4ns6phso6q17v07rj3rre9dm24e",
  "type":"sdnToOvn",
  "state": {
    "value":"scheduled",
    "description":""
  },
  "creation_timestamp":"2025-01-15T17:02:11.173068-05:00",
  "updated_timestamp":"2025-01-15T17:02:11.173068-05:00"
}
```